File: app/src/main/java/com/tonitealive/app/ui/views/signup/SignUpView.java
package com.tonitealive.app.ui.views.signup;

import com.tonitealive.app.ui.views.BaseView;


public interface SignUpView extends BaseView {
    String getUsername();
    String getEmail();
    String getPassword();
    String getConfirmPassword();

    void showProgressDialog();
    void hideProgressDialog();
}

File: app/src/main/kotlin/com/tonitealive/app/ui/views/signup/SignUpView.kt
package com.tonitealive.app.ui.views.signup

import com.tonitealive.app.ui.views.BaseView


interface SignUpView : BaseView {
    val username: String
    val email: String
    val password: String
    val confirmPassword: String

    fun showProgressDialog()
    fun hideProgressDialog()
}